hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/EstimatePrecompileFeature.java

```diff
@@ -1614,6 +1527,110 @@ public void exchangeRateTinyBarsToTinyCentsEstimateGas() {
         validateGasEstimation(data, EXCHANGE_RATE_TINYBARS_TO_TINYCENTS, estimatePrecompileContractSolidityAddress);
     }
 
+    // ETH_ESTIMATE-101
+    @Then("I call estimate gas that mints 1 FUNGIBLE token and returns the total supply and balance of treasury")
```

Comment:
nit: it would sound better in my opinion
```suggestion
    @Then("I call estimate gas that mints FUNGIBLE token and returns the total supply and balance of treasury")
```
Same applies for the test names below.